xray/utils/analyzermanager.go

```diff
@@ -232,7 +248,7 @@ func ExtractRelativePath(resultPath string, projectRoot string) string {
 
 func GetResultSeverity(result *sarif.Result) string {
 	if result.Level != nil {
-		if severity, ok := levelToSeverity[*result.Level]; ok {
+		if severity, ok := levelToSeverity[SarifLevel(*result.Level)]; ok {
```

Comment:
toLowercase?

Reply:
I will add just in case but it should return in lower